frontend/tests/setup.js

```diff
@@ -0,0 +1,5 @@
+import Vue from 'vue';
+import Vuetify from 'vuetify';
+
+Vue.use(Vuetify);
```

Comment:
This is already done in the file `plugins/vuetify.js`
Not sure if this is also part of the test setup...

Reply:
This is a part of the vue test utils setup because you need a local vuetify instance as documented on vuetify.
https://vuetifyjs.com/en/getting-started/unit-testing/

Reply:
makes sense!